generic_relations/relations.py

```diff
@@ -8,7 +9,17 @@
 from rest_framework import serializers
 
 
-class GenericRelatedField(serializers.Field):
+__all__ = ('GenericRelatedField',)
+
+
+class RenamedMethods(RenameMethodsBase):
+    renamed_methods = (
+        ('determine_deserializer_for_data', 'get_serializer_for_instance', UserWarning),
+        ('determine_serializer_for_data', 'get_deserializer_for_data', UserWarning),
+    )
```

Comment:
I'm not convinced this should be a `UserWarning` instead of a `DeprecationWarning`. Does Django use `UserWarning`?